requirements-test.txt

```diff
@@ -0,0 +1,2 @@
+django>=1.11,<2.1
+openwisp-utils>=0.2,<0.3
```

Comment:
this doesn't look right, here we need to put the dependencies for the automated test build, eg: https://github.com/openwisp/openwisp-controller/blob/master/requirements-test.txt